code/src/cljs/sixsq/nuvla/ui/apps/events.cljs

```diff
@@ -640,9 +641,12 @@
 
 (reg-event-fx
   ::paste-module
-  (fn [{{:keys [::spec/copy-module ::spec/module]} :db} [_ new-module-name]]
+  (fn [{{:keys [::spec/copy-module ::spec/module ::session-spec/session]} :db} [_ new-module-name]]
     (let [paste-parent-path (:path module)
+          user              (:user session)
           paste-module      (-> copy-module
+                                (dissoc :acl)
+                                (assoc-in [:acl :owners] [user])
```

Comment:
I think you don't even need to do that because api-server will do it for you on add